css/style.css

```
@@ -360,21 +334,26 @@ pre code {
 /* top button */
 
 .scroll #top {
-  opacity: .2;
+  border: 1px solid var(--border); 
+  opacity: .5;
+}
+
+#top:hover {
+  border: 2px solid var(--border); 

```

```suggestion
  border: 1px solid var(--border); 
```

I would suggest not increasing the border size. The effect it creates when hovering and expanding doesn't look natural.

---

Undid the changes back to 1px